pkg/cmd/server/api/validation/master.go

```diff
@@ -287,6 +287,18 @@ func ValidateAssetConfig(config *api.AssetConfig) fielderrors.ValidationErrorLis
 		allErrs = append(allErrs, urlErrs...)
 	}
 
+	if len(config.LoggingPublicURL) > 0 {
+		if _, loggingURLErrs := ValidateURL(config.LoggingPublicURL, "loggingPublicURL"); len(loggingURLErrs) > 0 {
```

Comment:
@sosiouxme Does the logging endpoint take an API token?

These probably need to be https anyway to avoid mixed content problems in the browser.